manifests/security/apparmor.pp

```diff
@@ -0,0 +1,14 @@
+# Since Apparmor is not enabled by default on Debian 9 we need to
+# change the boot parameters for the kernel to turn it on.
+# Debian 10 will, according to the current information, have
+# Apparmor enabled by default so this might not be needed in the future.
+# apparmor-utils is installed so we can test and debug profiles.
+class sunet::security::apparmor () {
+  package { 'apparmor-utils': ensure => latest }
+  if $::facts['operatingsystem'] == 'Debian' and $::facts['operatingsystemmajrelease'] == '9' {
```

Comment:
Minimun version to run the main branch is Debian 11 so we need no legacy support here.

Reply:
Solved by discussing on Slack.